aws/s3_types.go

```diff
@@ -23,6 +23,12 @@ func (bucket S3Buckets) MaxBatchSize() int {
 	return 500
 }
 
+// MaxConcurrentGetSize decides how many S3 buckets to fetch in one call.
+func (bucket S3Buckets) MaxConcurrentGetSize() int {
+	// To speed up bucket fetch part.
+	return 100
```

Comment:
Any particular reason for this number? Is there some AWS limit/throttle that drives this?

Reply:
No specific reason - I didn't want to execute too many goroutines e.g. if 5000 buckets are there - I didn't want to spawn 5000 goroutines and I did not know a right number - added 100 as a round figure.